k8s/operators/src/pool/diskpool/mod.rs

```diff
@@ -5,3 +5,4 @@
 pub(crate) mod client;
 /// The DiskPool custom resource definition.
 pub mod crd;
+pub mod new_crd;
```

Comment:
can we name these appropriately, v1beta1 and v1alpha1? 
Maybe we can have a crd folder, like so:
crd
-----mod.rs
-----v1alpha1.rs
-----v1beta1.rs

Reply:
Done